multiuser_posix.go

```diff
@@ -174,19 +153,19 @@ func MkdirAllTaskUser(dir string, perms os.FileMode) (err error) {
 	return nil
 }
 
-func makeFileOrDirReadWritableForUser(recurse bool, dir string, user *gwruntime.OSUser) ([]byte, error) {
+func makeFileOrDirReadWritableForUser(recurse bool, dir string, user *gwruntime.OSUser) error {
 	if recurse {
-		return exec.Command("/usr/sbin/chown", "-R", user.Name+":staff", dir).CombinedOutput()
+		return host.Run("/usr/sbin/chown", "-R", user.Name+":staff", dir)
 	}
-	return exec.Command("/usr/sbin/chown", user.Name+":staff", dir).CombinedOutput()
+	return host.Run("/usr/sbin/chown", user.Name+":staff", dir)
```

Comment:
The message here is that we've already logged the error, so we don't care about having stdout/stderr here - I like that, it seems like the right approach.

Reply:
Thanks!